docs/en/03-benchmark/benchmark.md

```diff
@@ -1061,6 +1090,35 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
     <td align="center">0.8497</td>
     <td align="center">0.8469</td>
     <td align="center"> - </td>
+    <td align="center">-</td>
+    <td align="center">-</td>
+  </tr>
+    <tr>
+    <td align="center" rowspan="2"><a href="https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/edsr/edsr_x2c64b16_g1_300k_div2k.py">EDSRx2</a></td>
```

Comment:
Why not test `EDSRx4` with ncnn-int8?

Reply:
sorry, I found edsr is x4 in benchmark.md after testing EDSRx2 with ncnn-int8. Converting model is a little slow. I will test EDSRx4 with ncnn-int8 in next week.

Reply:
have tested EDSRx4 and updated the result in this pr